meson.build

```diff
@@ -0,0 +1,277 @@
+project('libass', 'c', license: 'ISC',
+        meson_version: '>= 0.55.0',
+        # cpp_std specified because meson compiler options are currently global, can be set per-subproject with 0.63
+        default_options: ['c_std=c99', 'cpp_std=c++11', 'buildtype=debugoptimized', 'warning_level=2', 'default_library=static'],
```

Comment:
Why static by default? That's pretty unusual.

Reply:
There's discussion up-thread, but it's at the explicit request of the maintainers.

Reply:
@torque can you add a warning message when building non-static libass for non-Windows?
Something like
```
WARNING: this build does not properly support symbol visibility and the resulting shared lib is not suitable for distribution!
```